pyk/src/tests/test_kastManip.py

```diff
@@ -226,8 +227,12 @@ def test_simplify_bool(self) -> None:
         )
 
         for test_name, bool_in, bool_out in bool_tests:
-            bool_out_actual = simplify_bool(bool_in)
-            self.assertEqual(bool_out_actual, bool_out)
+            with self.subTest(i=test_name):
+                # When
+                bool_out_actual = simplify_bool(bool_in)
+
+                # Then
+                self.assertEqual(bool_out_actual, bool_out)
 
 
 class BuildClaimtest(TestCase):
```

Comment:
Are the `When/Then` enforced by `flake8`, or your own style choice?

Reply:
And I think you can just do `self.subTest(test_name)`, no need for `i=`?

Reply:
> Are the `When/Then` enforced by flake8, or your own style choice?

Personal preference. I like this [idiom](https://en.wikipedia.org/wiki/Given-When-Then) as it enforces a clean test structure.

Reply:
> And I think you can just do `self.subTest(test_name)`, no need for `i=`?

Indeed, good catch.